src/algebra/jordan/basic.lean

```diff
@@ -0,0 +1,213 @@
+/-
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import algebra.ring.basic
+import algebra.lie.of_associative
+import data.real.basic
+import linear_algebra.basic
+
+/-!
+# Jordan algebras
+
+Let `A` be a non-associative algebra (i.e. a module equipped with a bilinear multiplication
+operation). Then `A` is said to be a (commutative, linear) Jordan algebra if the multiplication is
+commutative and satisfies a weak associativity law known as the Jordan Identity: for all `a` and `b`
+in `A`,
+```
+(a * b) * a^2 = a * (b * a^2)
+```
+i.e. the operators of multiplication by `a` and `a^2` commute. Every associative algebra can be
+equipped with a symmetrized multiplication making it into a commutative Jordan algebra. Jordan
+algebras arising this way are said to be special. For an algebra `A` over a ring of characteristic
+not 2 this symmetrized multiplication may be defined, for `a` and `b` in `A`, as
+```
+a∘b = 1/2(a*b + b*a).
+```
+There are also exceptional Jordan algebras which can be shown not to be the symmetrization of any
+associative algebra. The 3x3 matrices of octonions is the canonical example.
+
+Every Jordan algebra `A` has a triple product defined, for `a` `b` and `c` in `A` by
+```
+⦃a b c⦄ = (a * b) * c - (a * c) * b + a * (b * c).
```

Comment:
Since `⦃a b c⦄` is not current notation for a triple product (nor is it likely to ever be due to clashes with binder notation), it might make sense to write this in `$$ $$`s instead for LaTeX formatting.